tokamak.py

```diff
@@ -13,6 +14,7 @@ def parse_args():
 	args = parser.parse_args()
 	return args
 
+
 def read_radial_build(file):
 	
 	radial_build = yaml.load(file, Loader=yaml.FullLoader)
```

Comment:
Variable name `file` is used by python. I'd suggest changing it to `file_`
https://dbader.org/blog/meaning-of-underscores-in-python